tests/scsi/007

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-3.0+
+# Copyright 2021 Google LLC
+#
+# Trigger the SCSI error handler.
+
+. tests/scsi/rc
+. common/scsi_debug
+
+DESCRIPTION="Trigger the SCSI error handler"
+
+QUICK=1
+
+requires() {
+	_have_scsi_debug
+}
+
+group_device_requires() {
+	true
+}
+
+config_hz() {
+	if [ -e /proc/config.gz ]; then
+		zcat /proc/config.gz
+	else
+		cat "/boot/config-$(uname -r)"
+	fi | sed -n 's/^CONFIG_HZ=//p'
+}
+
+trigger_scsi_eh() {
+	if ! _init_scsi_debug ""; then
+		return
+	fi
+
+	# Enable SCSI error handler logging
+	echo 63 > /sys/module/scsi_mod/parameters/scsi_logging_level
+
+	local dev freq jdelay target
+	for target in "${SCSI_DEBUG_TARGETS[@]}"; do
+		if ! dev=$(cd "/sys/class/scsi_device/${target}/device/block" &&
+			echo *); then
+			echo "Error: scsi_debug instance not found"
+			continue
+		fi
```

Comment:
You can replace all of this with `for dev in "${SCSI_DEBUG_DEVICES[@]}"`.

Reply:
Done.